uaa/slateCustomizations/source/index.html.md.erb

```diff
@@ -752,6 +752,104 @@ _Error Codes_
 | 403        | Forbidden - Insufficient scope (zone admins can only delete their own zone) |
 | 404        | Not Found - Zone does not exist                                             |
 
+## Creating a Key Provider Config
+
+A key provider config is created using a `POST` with a `KeyProviderConfig` object. An identifier will be generated for this object.
```

Comment:
Definitely need more explanation as to what a key provider config actually does with respect to integration with the customer's DCS service.